ml-conversational-analytic-tool/tests/test_commentAnalysis.py

```diff
@@ -0,0 +1,64 @@
+import unittest
+
+from ..commentAnalysis import CommentAnalyzer
+
+class TestCommentAnalysis(unittest.TestCase):
+
+    def test_CommentAnalyzer_init(self):
+        analyzer = CommentAnalyzer(['test'])
+        self.assertEqual({'test': 0}, analyzer.word_count,)
+
+    def test_analyzeComment(self):
+        analyzer = CommentAnalyzer(['test'])
+        result = analyzer.analyzeComment("This is a test PR")
+        self.assertEqual({'test': 1, 'Sentiment': 0.0, 'Code Blocks': 0}, result)
+
+    def test_analyzeComment_with_empty_comment(self):
+        analyzer = CommentAnalyzer(['empty comment'])
+        result = analyzer.analyzeComment("")
+        self.assertEqual({'empty comment': 0, 'Sentiment': 0.0, 'Code Blocks': 0}, result)
+
+    def test_analyzeComment_positive_comment(self):
```

Comment:
Noticed that some test names have the word 'with' and some do not. let's keep it consistent. I personally like adding 'with'.
```suggestion
    def test_analyzeComment_with_positive_comment(self):
```